src/Raven.Studio/typescript/common/shell/resourceSwitcher.ts

```diff
@@ -41,12 +45,18 @@ class resourceSwitcher {
         this.$selectDatabase = $('.form-control.btn-toggle.resource-switcher');
         this.$filter = $('.resource-switcher-container .database-filter');
 
+        let self = this;
```

Comment:
this self is not needed, since we use it in arrow function which captures context.